package-lock.json

```diff
@@ -0,0 +1,5105 @@
+{
+  "name": "poc-network",
```

Comment:
Currently, I have it set up with `pnpm`instead of `npm`. Happy to switch through. Thoughts? cc @kfaria

Reply:
oooo i havent heard of `pnpm` which is which why didnt use it! i can switch to keep thinks consistent- the `pnpm-lock.yaml` file makes sense now though 😅

Reply:
@shamoons installed with pnpm, package-lock no longer exists 🙏